actions/build-push-to-dockerhub/action.yaml

```diff
@@ -25,12 +25,18 @@ inputs:
     description: |
       The dockerfile to use.
     required: false
+  clean_repo:
+    description: |
+      Whether to run `git clean -ffdx && git reset --hard HEAD`.
+    default: "true"
```

Comment:
you said the default was `false` up above in the readme, but here it is `true` :slightly_smiling_face: